### DISCUSSION TITLE
Soroban HelloWorld smart contract NFT

### DISCUSSION THREAD

**Discussion by LorDDark6660** [stellar/sorobanathon#52] [General]
Hello world smart contract on NFT :)

![hw2](https://user-images.githubusercontent.com/92062140/202852657-7798c112-51b3-4f66-910e-744d6afe975d.gif)

https://litemint.com/items/GBN3ANAIDAE24WQDCTAAWBYCHU7HBADIXO5HAYSB4E5EJESCY2GC32IC/HelloWorld

**Comment by stellarjohn** [comment]
Howdy LorDDark6660,

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 300 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.
Please let us know if you have any questions.

Cheers!

**Reply by LorDDark6660** [reply]
Done, thank you ✨
